packages/entity-detail/src/util/detailView/reduceInitialValues.spec.js

```diff
@@ -0,0 +1,85 @@
+import reduceInitialValues, {getRelevantAttributes} from './reduceInitialValues'
+
+const formValues = {
```

Comment:
a good tests should also contain data that is not be affected by your changes. e.g. a string field